start R6 document so that we have a place to add items as development…

fkr opened the pull request:
… progresses

fkr committed:
start R6 document so that we have a place to add items as development progresses

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
start R6 document so that we have a place to add items as development progresses

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
start adding more contents for R6

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
start adding more contents for R6

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

jschoone committed:
Update Release6.md

Signed-off-by: Jan Schoone <6106846+jschoone@users.noreply.github.com>

jschoone committed:
Update Release6.md

Signed-off-by: Jan Schoone <6106846+jschoone@users.noreply.github.com>

fkr commented on Release6.md:
```diff
@@ -55,6 +75,9 @@ Overall with Release 6 it becomes apparent that SCS is efficient to operate.
 ## List of known issues & restrictions in R6
 
+### KaaS
+* Some features of KaaS v1 are not available yet in KaaS v2 because they are WIP in upstream CAPO
```
Would it make sense to list these features here?

fkr committed:
Openstack -> OpenStack

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
Openstack -> OpenStack

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
Mention OVS and OVN excplicitly

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
Mention OVS and OVN excplicitly

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
Reword and mention Ceph Reef

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
Reword and mention Ceph Reef

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

markus-hentsch committed:
Add preview for Domain Manager Persona

Signed-off-by: Markus Hentsch <markus.hentsch@cloudandheat.com>

markus-hentsch committed:
Add preview for Domain Manager Persona

Signed-off-by: Markus Hentsch <markus.hentsch@cloudandheat.com>

markus-hentsch committed:
Add preview for domain list API scope fix

Signed-off-by: Markus Hentsch <markus.hentsch@cloudandheat.com>

markus-hentsch committed:
Add preview for domain list API scope fix

Signed-off-by: Markus Hentsch <markus.hentsch@cloudandheat.com>

markus-hentsch commented:
I added the domain related work as preview sections.

PixelPhantomX committed:
Started adding Zuul details to Release6.md

Signed-off-by: Roy Farrell <farrell@gonicus.de>

PixelPhantomX committed:
Started adding Zuul details to Release6.md

Signed-off-by: Roy Farrell <farrell@gonicus.de>

matofeder committed:
Add Observability release notes (#18)

Signed-off-by: Matej Feder <matej.feder@dnation.cloud>

matofeder committed:
Add Observability release notes (#18)

Signed-off-by: Matej Feder <matej.feder@dnation.cloud>

garloff committed:
Octavia bugs, why again KaaSv1, os-hm improvements, SecFixes, Thanks

So we fill out some sections:
* Known limitations/issues: octavia bugs
* Explain why we do KaaSv1 once more and repeat that it's deprecated and
  will be removed
* Mention os-hm (probably a last time)
* Link blog articles on Sec Fixes
* Explicitly thank PS: They have been supporting us well with testing

Signed-off-by: Kurt Garloff <kurt@garloff.de>

garloff committed:
Octavia bugs, why again KaaSv1, os-hm improvements, SecFixes, Thanks

So we fill out some sections:
* Known limitations/issues: octavia bugs
* Explain why we do KaaSv1 once more and repeat that it's deprecated and
  will be removed
* Mention os-hm (probably a last time)
* Link blog articles on Sec Fixes
* Explicitly thank PS: They have been supporting us well with testing

Signed-off-by: Kurt Garloff <kurt@garloff.de>

garloff committed:
Add deprecations, docuemtnation and standards.

Signed-off-by: Kurt Garloff <kurt@garloff.de>

garloff committed:
Add deprecations, docuemtnation and standards.

Signed-off-by: Kurt Garloff <kurt@garloff.de>

fkr committed:
remove this sentence, this is not the case.

discussed with @garloff

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
add paragraph for Release 6

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
mention k3s on the controlplane of the iaas reference implementation

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
consistent spacing

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
explicitly name the tang role deprecation in osism

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr committed:
mention `osism manage image octavia` in the operator highlights

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

joshmue committed:
Add Central API release notes

Signed-off-by: Joshua Mühlfort <muehlfort@gonicus.de>

joshmue commented:
Added a section about the central API.

reqa committed:
Add IAM release notes

Signed-off-by: Arvid Requate <requate@univention.de>

bitkeks committed:
Add highlighted blog posts to documentation in R6 rel.notes

Signed-off-by: Dominik Pataky <pataky@osb-alliance.com>

bitkeks committed:
Add security/pentesting notes to R6 release notes

Signed-off-by: Dominik Pataky <pataky@osb-alliance.com>

bitkeks committed:
Re-phrase security notes

Signed-off-by: Dominik Pataky <pataky@osb-alliance.com>

fkr committed:
reword, since the now released observability stack does NOT replace the openstack health monitor.

The new OpenStack health monitor is being developed as part of VP12, which just started.

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

fkr commented on Release6.md:
I'm not really happy with this. These are technical release notes and we should focus on the stuff we are releasing. This seems good content for an outlook blogpost imho.

garloff replied:
Well, maybe we shorten this and attach it to the section that talks about the pentesting that already happened and that improved R6 security?

garloff committed:
Spellchecking.

Signed-off-by: Kurt Garloff <kurt@garloff.de>

garloff committed:
Spellchecking.

Signed-off-by: Kurt Garloff <kurt@garloff.de>

garloff committed:
Merge branch 'feat/r6' of github.com:SovereignCloudStack/release-notes into feat/r6

Signed-off-by: Kurt Garloff <kurt@garloff.de>

garloff committed:
Make pipeline-driven security an addendum to pentesting section.

This reflects that we keep forward-looking statements short in our
Release Notes. (And maybe we wnat to publish a complete blog article
here on our vision and plans?)

Also remove the header that marks the R6 release notes as pre-release
notes.

Signed-off-by: Kurt Garloff <kurt@garloff.de>

garloff committed:
More spellchecking.

Signed-off-by: Kurt Garloff <kurt@garloff.de>

garloff reviewed (approved):
I think this is in a reasonable shape now.
